README.md

```diff
@@ -69,6 +69,7 @@ Offical Website: [tdengine.com](https://tdengine.com)
 - [taos-rs](https://github.com/yuerrd/taos-rs) - TDengine Java RESTful Client.
 - [TDengine Rust bindings](https://github.com/songtianyi/tdengine-rust-bindings) - It's a Rust bindings project for TDengine.
 - [TDenginex](https://github.com/lizhaochao/TDenginex) - TDengine Connector for Elixir.
+- [DBUtility.TDengine](https://github.com/cockroach888/GSA.MOLLE.ToolKits/tree/main/src/ToolKits.DBUtility/ToolKits.DBUtility.TDengine/src) - C# Connector for TDengine. (.net6+ [NuGet package](https://www.nuget.org/packages/DBUtility.TDengine))
```

Comment:
1. Url should link to repo's home page, such as https://github.com/cockroach888/GSA.MOLLE.ToolKits.
2. Should not add additional link(NuGet package) in description.

Reply:
Related issues have been addressed.